pages.sv/windows/print.md

```diff
@@ -0,0 +1,12 @@
+# print
+
+> Skriv ut en textfil till en skrivare.
+> Mer information: <https://learn.microsoft.com/windows-server/administration/windows-commands/print>.
+
+- Skriv ut en textfil till standardskrivaren:
+
+`print {{path/to/file}}`
```

Comment:
Please translate the placeholders, e.g. `path/to/file`, `user`, `computer` etc., for get this easier for non-English speakers.

Reply:
Sorted all now i think =)